ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -366,6 +374,7 @@ export default defineComponent({
 
     const scrollToFirstError = async () => {
       setTimeout(() => {
+        document.getElementsByClassName('border-error-left').length > 0 &&
```

Comment:
Safety check, was throwing errors once `document.getElementsByClassName('border-error-left')` was empty/errors cleared.

Reply:
Yea, I was hoping a timeout would prevent it, but this is another useful check.